autoload/codefmt/formatterhelpers.vim

```diff
@@ -90,18 +90,21 @@ endfunction
 " @throws ShellError if the {cmd} system call fails
 " @throws WrongType
 function! codefmt#formatterhelpers#AttemptFakeRangeFormatting(
-    \ startline, endline, cmd) abort
+    \ startline, endline, cmd, ignoreerrors, skipfirstnlines) abort
   call maktaba#ensure#IsNumber(a:startline)
   call maktaba#ensure#IsNumber(a:endline)
 
   let l:lines = getline(1, line('$'))
   let l:input = join(l:lines[a:startline - 1 : a:endline - 1], "\n")
 
-  let l:result = maktaba#syscall#Create(a:cmd).WithStdin(l:input).Call()
+  let l:result = a:ignoreerrors ?
+      \ maktaba#syscall#Create(a:cmd).WithStdin(l:input).Call(0)
+      \ : maktaba#syscall#Create(a:cmd).WithStdin(l:input).Call() 
```

Comment:
Instead of this big ternary you can just pass in !a:ignoreerrors at the end: `let l:result = maktaba#syscall#Create(a:cmd).WithStdint(l:input).Call(!a:ignoreerrors)`.

Reply:
Facepalm. Done.